### PR TITLE
Reformatted README

### PR DESCRIPTION
**Pull request by Odizinne**
Done a bit of reformatting for the readme for clarity.

Feel free to put it to trash if you don't like it!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,30 +3,32 @@ This is an app to play Pokerogue.net in an app window. Wow!
...
+- Clone this repository with git or download the source code as zip
+- Install [NodeJS](https://nodejs.org/en)
+- Open a command prompt to the directory you stick this repo in
+- Install electron: `npm install electron --save-dev` 
+- Install the electron builder `npm install electron-builder --save-dev`
+
+Run from command prompt with `npm start` or build the full app with `npm run build:platform`.<br/>
+Each version has to be built from its own OS for it to work properly when zipped up (thanks Electron!)
 
 Play! The app is found in dist/unpacked/Pokerogue.exe if you built it that way.
```
Comment (SavGRY) [inline README.md:34]: I think that you could also format  `dist/unpacked/Pokerogue.exe`. Maybe you just forgot :)

**Review by SavGRY — COMMENTED** [review]
Just a minor nitpick

File: README.md (hunk 1 above)
Comment (Admiral-Billy) [reply README.md:34]: This seems valid to me!

**Comment by Admiral-Billy** [comment]
Thanks for this :)

**Pull request merged**